contrib/babelfishpg_tsql/antlr/TSqlParser.g4

```diff
@@ -3436,10 +3436,7 @@ constant_LOCAL_ID
 // https://docs.microsoft.com/en-us/sql/t-sql/language-elements/expressions-transact-sql
 // Operator precendence: https://docs.microsoft.com/en-us/sql/t-sql/language-elements/operator-precedence-transact-sql
 expression
-    : local_id (DOT calls+=method_call)*                                        #local_id_expr
```

Comment:
is there potential side effects from altering parser definition of expression like this, and is it necessary? IMO feels invasive

Reply:
No there are no side effects for this change as we have just created an umbrella rule to handle CLR_UDT types, everything apart from them will work as before and it is necessary as without it we will be handling the functions with tagName which can be a concern